js_basics_functions/task2.js

```diff
@@ -0,0 +1,5 @@
+const validateAge = age => age >= 18;
```

Comment:
much better, but this name does not allow me me to understand that this function returns boolean value